composer.json

```diff
@@ -15,6 +15,9 @@
         "php-parallel-lint/php-console-highlighter": "^1.0",
         "php-parallel-lint/php-parallel-lint": "^1.3.1",
         "phpcompatibility/php-compatibility": "dev-develop",
+        "phpstan/extension-installer": "^1.4.3",
+        "phpstan/phpstan": "^1.12.26",
+        "szepeviktor/phpstan-wordpress": "^v1.3.5",
```

Comment:
Since it is a new addition, it'd be better to start with 2.0?!

Reply:
We can't, because we currently require PHP 7.2+, but v2 requires PHP 7.4+

Reply:
ah, ok. Makes sense.